docs/doc/15-sql-functions/90-context-functions/index.md

```diff
@@ -0,0 +1,10 @@
+---
+title: 'Context Functions'
+---
+
+| Function                     | Description                                                                                                                                                                                                                      | Example                              | Result                                                                                  |
+|------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------|-----------------------------------------------------------------------------------------|
+| **CONNECTION_ID()**          | Return the connection id for the connection.                                                                                                                                                                                     | **CONNECTION_ID()**                  | 9b16ca73-e393-43bb-a77b-fa546e1b7b69                                                    |
+| **current_user()**           | Returns the user name and host name combination for the account that the server used to authenticate the current client. This account determines your access privileges. The return value is a string in the utf8 character set. | **current_user()**                   | 'root'@'127.0.0.1'                                                                      |
```

Comment:
This function name better to use capital letters like other functions.

Reply:
I agree. I initiated a discussion in the tracking issue to address this.

Reply:
Updated.